.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         run: npm run lint:style
   playwright:
     name: Accessibility and Unit Tests
-    timeout-minutes: 10
+    timeout-minutes: 15
```

Comment:
I've re-run this many times locally and in CI.
- Locally, these changes are 1.1x (aXe) to 1.5x (unit) slower, which is barely noticeable
- In CI however, these changes are 1.3x (aXe) to 1.8x (unit) slower, which is adding ~3.5 minutes to this job. It's still quicker than the visual-diff tests though! 😅 

It's admittedly not great, but also not shocking since we're now `await`-ing all nested Lit components for ~1650 tests, and doing a lot more resetting of the browser state between tests. If the slowness becomes a problem, there's a lot of tests using timeouts that we could look into fixing up.